tiatoolbox/utils/misc.py

```diff
@@ -511,3 +515,42 @@ def assert_dtype_int(input_var, message="Input must be integer."):
     """
     if not np.issubdtype(np.array(input_var).dtype, np.integer):
         raise AssertionError(message)
+
+
+def download_data(url, save_path, overwrite=False):
+    """Download data from a given URL to location. Can overwrite data if demanded
+    else no action is taken
+
+    Args:
+        url (path): URL from where to download the data.
+        unzip_dir (str): location to unzip the data.
+    """
```

Comment:
empty line at end of multiline docstring.

Reply:
Done